src/assets/scss/components/statusContainer.scss

```diff
@@ -0,0 +1,106 @@
+.status-container {
+  background-color: $off-white;

```

Comment:
let's use pure white (`$eos-white`) for the background and let's remove the off-white variable as we won't be using it elsewhere

Reply:
same as above:
> it seems that you removed the entire background for the columns. can you please revert this change back and just add the changes suggested? it will be easier to visualize as columns.

Reply:
@cyntss It is looking something like this on replacing with total white. Is it good to go?

![localhost_3000_ (2)](https://user-images.githubusercontent.com/75155230/190513963-10c8e98e-fc4e-4c66-b3e5-a9a815e47ff4.png)

Reply:
Should I add some shadow for each story card?

Reply:
this looks perfect @kailash360